internal/commands/vaultsecrets/secrets/displayer.go

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package secrets
+
+import (
+	models "github.com/hashicorp/hcp-sdk-go/clients/cloud-vault-secrets/stable/2023-06-13/models"
+	"github.com/hashicorp/hcp/internal/pkg/format"
+)
+
+type displayer struct {
+	secrets []*models.Secrets20230613Secret
+	single  bool
+}
+
+func newDisplayer(single bool, secrets ...*models.Secrets20230613Secret) *displayer {
+	return &displayer{
+		secrets: secrets,
+		single:  single,
+	}
+}
+
+func (d *displayer) DefaultFormat() format.Format {
+	return format.Table
+}
+
+func (d *displayer) Payload() any {
+	if d.single {
+		if len(d.secrets) != 1 {
+			return nil
+		}
+
+		return d.secrets[0]
+	}
+
+	return d.secrets
+}
+
+func (d *displayer) FieldTemplates() []format.Field {
+	return []format.Field{
+		{
+			Name:        "Secret Name",
+			ValueFormat: "{{ .Name }}",
+		},
+		{
+			Name:        "Latest Version",
+			ValueFormat: "{{ .LatestVersion }}",
+		},
+		{
+			Name:        "Created At",
+			ValueFormat: "{{ .CreatedAt }}",
+		},
+	}
```

Comment:
Just curious - are these the fields we return after secret creation with vlt? Wondering if we need `CreatedAt`

Reply:
vlt doesn't support this format today on this command. however, if you try secrets read you will see the tabular form. I am hoping our internal beta will provide more feedback on wha to keep and drop down the line. for now, mostly gut feeling.

Reply:
@mercedesbh and I discussed internally to align on a common pattern.